waku/v2/node/wakunode2.nim

```diff
@@ -433,7 +434,7 @@ proc mountFilter*(node: WakuNode, filterTimeout: Duration = WakuFilterTimeout) {
     info "push received"
     for message in msg.messages:
       node.filters.notify(message, requestId) # Trigger filter handlers on a light node
-      waku_node_messages.inc(labelValues = ["filter"])
+      waku_node_messages.inc(labelValues = ["filter", message.contentTopic])
```

Comment:
What if contentTopic isn't filled out? IIRC it is a SHOULD, do we want to assert this?

Reply:
Good point. It will write an empty string to Prometheus, which I imagined would still be query-able, but [quick search](https://github.com/grafana/grafana/issues/19410) makes it seem like it won't be straightforward to extract label values. Probably best to add a placeholder label for messages without content topic, like `"none"`.

Reply:
Addressed in ae3e1cc